download.html

```diff
@@ -23,38 +23,34 @@
             </div>
     </div>
         </section>
-	<section id="configure">
-            <div class="text-center">
-                <h1>Configure</h1>
+
+<section id="requirements" class="container">
+    <div class="row ptb-15">
+        <div class="col-xs-12">
```

Comment:
This Requirements section seems a bit funny. There is a lot of dead space on the right hand side. Does it need to be in a column?

Reply:
It might look better if centered.